streams/src/test/java/org/apache/kafka/streams/integration/TaskAssignorIntegrationTest.java

```diff
@@ -89,7 +89,7 @@ public void shouldProperlyConfigureTheAssignor() throws NoSuchFieldException, Il
                 mkEntry(StreamsConfig.MAX_WARMUP_REPLICAS_CONFIG, "7"),
                 mkEntry(StreamsConfig.PROBING_REBALANCE_INTERVAL_MS_CONFIG, "480000"),
                 mkEntry(StreamsConfig.InternalConfig.ASSIGNMENT_LISTENER, configuredAssignmentListener),
-                mkEntry(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, 9)
+                mkEntry(AdminClientConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, 90_000)
```

Comment:
Just curious, why the change from 9 to 90,000?

Reply:
For fun

Reply:
I mean, because the value has to be larger than the `request.timeout.ms` which defaults to 30,000

Reply:
Ah, thanks.